securitas/templates/password-reset.html

```diff
@@ -22,24 +24,20 @@
               <span class="card-title">Password Reset</span>
 
               <div class="input-field col s12">
-                <input placeholder="jdoe" id="username" name="username" type="text" class="validate" tabindex="1">
-                <label for="username">Username</label>
+                {{ macros.with_errors(password_reset_form.username, class="validate", placeholder="jbloggs", tabindex="1")}}
               </div>
 
               <div class="input-field col s12">
-                <input placeholder="current_password" id="current_password" name="current_password" type="password" class="validate" tabindex="2">
-                <label for="current_password">Current Password</label>
+                {{ macros.with_errors(password_reset_form.current_password, class="validate", placeholder="current password", tabindex="2")}}
               </div>
 
               <div class="input-field col s12">
-                <input placeholder="password" id="password" name="password" type="password" class="validate" tabindex="3">
-                <label for="password">New Password</label>
+                {{ macros.with_errors(password_reset_form.password, class="validate", tabindex="3")}}
               </div>
 
               <div class="input-field col s12">
-                <input placeholder="password" id="password_confirm" name="password_confirm" type="password" class="validate" tabindex="4">
-                <label for="password_confirm">Password Confirmation</label>
-              </div>
+                {{ macros.with_errors(password_reset_form.password_confirm, class="validate", tabindex="4")}}
+                </div>
 
               <div class="input-field col s12">
                 <button class="btn waves-effect waves-light light-blue darken-4 w100" id="submit" type="submit" tabindex="5">Submit</button>
```

Comment:
You'll also need to add the macro that displays the `non_field_errors` errors, so something like:
```
{{ macros.non_field_errors(password_reset_form) }}
```